WordPress/src/test/java/org/wordpress/android/ui/comments/usecases/PaginateCommentsUseCaseTest.kt

```diff
@@ -8,16 +8,13 @@ import com.nhaarman.mockitokotlin2.whenever
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.launch
-import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.test.runBlockingTest
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Before
-import org.junit.Rule
 import org.junit.Test
 import org.mockito.Mock
-import org.mockito.Mockito
+import org.mockito.Mockito.*
```

Comment:
Hey @ParaskP7 👋🏼 
❓ Is the wild cart import necessary?.

Reply:
👋 @AjeshRPai and thanks for noticing and hence the comment! 🙇

Actually no, I can replace that with `import org.mockito.Mockito.'when'`, maybe that's better, although, to be honest I don't like the `'when'` import syntax and in general us needing this `'when'` syntax in tests. PS: This `*` import was done automatically for me by AS, when I clicked on optimizing the import.

Since, I am not seeing any such `import org.mockito.Mockito.*` imports for Mockito in our project, I'll go ahead and make this change, especially since we generally don't like `*` imports. 👍

Reply:
FYI: This is now done: 42b38e81ac006fea92b99e5da3824c9015abb891

Reply:
Thanks, @ParaskP7  👍🏼 .